elements/pfe-label/pfe-label.ts

```diff
@@ -0,0 +1,64 @@
+import type { TemplateResult } from 'lit';
+
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { pfelement } from '@patternfly/pfe-core/decorators.js';
+
+import '@patternfly/pfe-icon';
+
+import styles from './pfe-label.scss';
+
+/**
+ * Label
+ * @slot - Place element content here
```

Comment:
a prescriptive explanation would be better here
```suggestion
 * @slot - Label text
```

Reply:
Fixed